lib/faraday/multipart.rb

```diff
@@ -5,6 +5,8 @@
 require_relative 'multipart/middleware'
 require_relative 'multipart/version'
 
+require 'multipart/post/version'
```

Comment:
Q: can this fail with a LoadError on the earlier version? 

That LoadError could be caught and explained.

Reply:
I'll check if the old release had that there.

Reply:
https://github.com/socketry/multipart-post/commit/49584cf671792a4fb6de0a8e2241f00f08c06334
This shows the before and after of the version.rb - where the constant was before, and where it is now.

Reply:
The current changeset no longer has a `require 'multipart/post/version'` in multipart.rb, only in version.rb, where we grab the LoadError and load the old files.